src/components/appointment/AppointmentHostCard.tsx

```diff
@@ -0,0 +1,98 @@
+import { AppointmentDetail } from "@/types/Appointment";
+import { CardContent, Grid, Stack, Typography, Box, Chip } from "@mui/material";
+import Image from "next/image";
+import BorderWithShadow from "../public/BorderWithShadow";
+import AppointmentHeader from "./AppointmentHeader";
+
+const chipsStyle = {
+  minWidth: 100,
+  height: 40,
+  fontSize: 18,
+  fontWeight: 'bold',
+  border: "black solid 4px",
+  borderRadius: "16px",
+  boxShadow: "4px 4px #BFBFBF",
+}
+type props = {
+  appointmentDetail: AppointmentDetail,
+};
+export default function AppointmentHostCard(props: props) {
+  return <Grid container
+    spacing="40px"
+    width="80vw"
+  >
+    <Grid item xs={12} md={6} style={{ display: "flex" }}>
+      <BorderWithShadow>
+        <CardContent>
+          <Stack spacing="12px" sx={{ padding: "20px", height: "100%" }}>
+            <AppointmentHeader detailHeader={props.appointmentDetail.detailHeader} isHost={true} />
+            <Box >
+              <Typography variant="h5" sx={{ fontWeight: "bold", marginBottom: "4px" }}>Image</Typography>
+              <Grid container spacing={1}>
+                {props.appointmentDetail.images.map((e, index) => (
+                  <Grid item key={'i' + index}>
+                    <Image src={e} alt="location" width={150} height={150} />
+                  </Grid>
+                ))}
+              </Grid>
```

Comment:
I think the third image could fit in the same line as image 1 and 2 since enough space is available (also according to design)
<img width="499" alt="image" src="https://user-images.githubusercontent.com/62459518/222942020-cb252bb3-e1e9-40e8-80cd-8db01bac8cf3.png">

Reply:
i reduce image size down to 140